Packs/AWS-SNS/Integrations/AWSSNS/AWSSNS.yml

```diff
@@ -0,0 +1,284 @@
+category: IT Services
+commonfields:
+  id: AWS - SNS
+  version: -1
```

Comment:
The integration image AWSSNS_image.png dimensions are 360x230. The requirements are 120x50.

Reply:
@ilappe I think the new one is the correct one. Not so sure the resolution part. Please help to check. Thanks.

Reply:
The image dimensions are 75x50. The requirements are 120x50.
 so the build still fail on that

Reply:
@ilappe I push the new one to exactly match the 120x50. Please take a look. thanks.